src/views/AddLead/requests.js

```diff
@@ -149,6 +149,7 @@ const requestOptions = {
         method: methods.POST,
         body: ({ params }) => ({
             ...params.values,
+            authors: isDefined(params.values.authors) ? params.values.authors : [],
```

Comment:
Why wasn't author's default values set to []?

Reply:
easier to handle it at one point instead of multiple

Reply:
Actually, it's easier if you do it during initialization. An empty array can never be changed to undefined using MultiSelectInput.
But, should look into faram behavior

Reply:
While getting the values from redux, if the url is not present it returns an empty object, the best place would be to do it.

Reply:
why not add { author: [] } in default value in redux selector?